DOCUMENTATION.md

```diff
@@ -9,6 +9,8 @@
         -   [Pull package for EMS Store](#pull-package-for-ems-store)
     -   [Pull/Push individual assets from/to Studio](#pullpush-individual-assets-fromto-studio)
     -   [Overwrite Package In Studio](#overwrite-package-in-studio)
+    -   [Export multiple packages at once from Studio](#export-multiple-packages-at-once-from-studio)
```

Comment:
I think a more correct term would be batch export and import. Let's replace it everywhere in the docs.